languages/en.ini

```diff
@@ -1122,6 +1122,8 @@ Production Credits = "Production Credits"
 Profile = "Profile"
 profile_update = "Your profile was updated as requested"
 pronounced = "pronounced"
+proquestfsg_recommendations = "ProQuest Results"
+proquestfsg_recommendations_more = "More ProQuest Results"
```

Comment:
I don't think the typical patron would have any idea what "ProQuest" means, so this will likely have to be templated, but I can't think of any better default.